test/functional/admin/worldwide_offices_controller_test.rb

```diff
@@ -292,8 +292,8 @@ class Admin::WorldwideOfficesControllerTest < ActionController::TestCase
               contact_numbers_attributes: {
                 "0" => {
                   id: contact_number.id,
-                  label: contact_number.label,
-                  number: contact_number.number,
+                  label: "",
+                  number: "",
```

Comment:
It appears these values aren't being included at all in the payload, hence the `undefined method `empty?' for nil:NilClass` error.  So both these lines can just be deleted.

Reply:
Sorted :)